apps/examples/lcd_test/example_lcd.c

```diff
@@ -371,6 +419,12 @@ int lcd_test_main(int argc, char *argv[])
 	int fd = 0;
 	int p = 0;
 	char port[20] = { '\0' };
+#ifdef CONFIG_INPUT_IST415
```

Comment:
This file is not specific touch IC.
Please use common touch config instead of CONFIG_INPUT_IST415 here